bundles/org.openhab.binding.ojelectronics/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,165 @@
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+		<label>OJ Electronics OWD5/MWD5 Thermostat</label>
+		<description>OJ Electronics OWD5/MWD5 Thermostat</description>
+		<category>RadiatorControl</category>
+		<channels>
+			<channel id="floorTemperature" typeId="floorTemperature" />
+			<channel id="roomTemperature" typeId="roomTemperature" />
+			<channel id="groupName" typeId="groupName" />
+			<channel id="groupId" typeId="groupId" />
+			<channel id="online" typeId="online" />
+			<channel id="heating" typeId="heating" />
+			<channel id="thermostatName" typeId="thermostatName" />
+			<channel id="regulationMode" typeId="regulationMode" />
+			<channel id="serialNumber" typeId="serialNumber" />
+			<channel id="comfortSetpoint" typeId="comfortSetpoint" />
+			<channel id="comfortEndTime" typeId="comfortEndTime" />
+			<channel id="boostEndTime" typeId="boostEndTime" />
+			<channel id="manualModeSetpoint" typeId="manualModeSetpoint" />
+			<channel id="vacationEnabled" typeId="vacationEnabled" />
+		</channels>
+		<config-description>
+			<parameter name="serialNumber" type="text" required="true">
+				<label>Serial Number</label>
+				<description>Serial number of the thermostat. You can find the serial number in the app or on the thermostat itself.</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+	<channel-type id="floorTemperature">
+		<item-type>Number</item-type>
+		<label>Floor temperature</label>
+		<category>Temperature</category>
+		<state pattern="%.1f °C" readOnly="true" />
+	</channel-type>
+	<channel-type id="groupName">
+		<item-type>String</item-type>
+		<label>Group name</label>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="groupId">
+		<item-type>Number</item-type>
+		<label>Group Id</label>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="online">
+		<item-type>Switch</item-type>
+		<label>Online</label>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="heating">
+		<item-type>Switch</item-type>
+		<label>Heating</label>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="roomTemperature">
+		<item-type>Number</item-type>
+		<label>Room temperature</label>
+		<category>Temperature</category>
+		<state pattern="%.1f °C" readOnly="true" />
+	</channel-type>
+	<channel-type id="thermostatName">
+		<item-type>String</item-type>
+		<label>Thermostat name</label>
+		<state readOnly="true" />
+	</channel-type>
+	<channel-type id="regulationMode">
+		<item-type>String</item-type>
+		<label>Regulation mode</label>
+		<state readOnly="true">
+			<options>
+				<option value="1">Auto</option>
+				<option value="2">Comfort</option>
+				<option value="3">Manual</option>
+				<option value="4">Vacation</option>
+				<option value="6">Frost protection</option>
+				<option value="8">Boost</option>
+				<option value="9">Eco</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="serialNumber">
```

Comment:
Is it necessary to expose the serialNumber as a Channel, since it is already a property.